nengo_gui/gui.py

```diff
@@ -100,9 +105,9 @@ def _confirm_shutdown(self, signum, frame):
             if line[0].lower() == 'y':
                 raise ServerShutdown()
             else:
-                print("Resuming...")
+                logger.info("Resuming...")
         else:
-            print("No confirmation received. Resuming...")
```

Comment:
Actually, I might argue that the other `print`s in this function should stay `print`s too.  Though it is not as clear cut for those.